app/models/laptop.rb

```diff
@@ -0,0 +1,8 @@
+class Laptop < ApplicationRecord
+  has_many :review
+  before_create :slugify
+
+  def slugify
+    self.slug = name.parameterize
```

Comment:
This method might be better in a helper file (in the helpers folder) as it has something to do more with presentation than data manipulation. ActiveRecord is the place where we do queries, validations, `before`, `after` callbacks, etc which are all involved in manipulating and reading data

Reply:
As mentioned, this method isn't right for the model. Anyway it's an optional change but it will help you in the future to take note of this